src/platform/Ameba/CHIPDevicePlatformConfig.h

```diff
@@ -67,3 +67,7 @@
 #define CHIP_DEVICE_CONFIG_USE_TEST_SETUP_DISCRIMINATOR 0xF00
 
 #define CONFIG_RENDEZVOUS_MODE 6
+
+#define CHIP_DEVICE_CONFIG_ENABLE_EXTENDED_DISCOVERY 1
```

Comment:
Should this be the platform default?  This is only needed for devices that want to do UDC.

Reply:
You are right, I will remove this from here. Thanks